web/src/components/users/FirstUser.jsx

```diff
@@ -220,11 +223,44 @@ export default function FirstUser() {
   const submitDisable = formValues.userName === "" || (isSettingPassword && !usingValidPassword);
 
   const displaySuggestions = !formValues.userName && formValues.fullName && showSuggestions;
+  useEffect(() => {
+    if (displaySuggestions) {
+      setFocusedIndex(-1);
+      setSuggestions(suggestUsernames(formValues.fullName));
+    }
+  }, [displaySuggestions, formValues.fullName]);
+
   const onSuggestionSelected = (suggestion) => {
     setInsideDropDown(false);
     setFormValues({ ...formValues, userName: suggestion });
```

Comment:
One thing more.

While doing a manual test, I've realized that the focus is now lost when a suggestion is chosen by clicking on it. It works like a charm if using the keyboard instead.

I think would be enough to create a `ref` (let's say, usernameInputRef) and use it to ensure the focus comes back here again.  Just saying, maybe there is a better, more elegant solution.

```suggestion
    setFormValues({ ...formValues, userName: suggestion });
    usernameInputRef.current?.focus()
```